web/themes/custom/itkdev/itkdev_project_theme/templates/reports-project-track.html.twig

```diff
@@ -20,5 +20,9 @@
         <div class="w-full mb-16"><canvas id="track-type-{{ key }}" class="chart" track-id="{{ key }}"></canvas></div>
       {% endif %}
     {% endfor %}
+
+    <div>
+      <a href="javascript:history.back()" class="inline-block btn-primary bg-black text-white hover:bg-stone-700">{{ 'Go Back'|t }}</a>
+    </div>
```

Comment:
Would it make more sense to go (back) to the report for the track? Using `history.back()`can send the user back to something outside the system and that's probably not what the user wants.

Reply:
you can get to this page through reports as well as from the track so i think history.back is the best solution.